ext/BraketSimulatorBraketExt/BraketSimulatorBraketExt.jl

```diff
@@ -154,6 +154,13 @@ end
 Base.convert(::Type{Braket.AbstractProgram}, p::BraketSimulator.Program) = Braket.Program(Braket.braketSchemaHeader("braket.ir.jaqcd.program", "1"), [convert(Braket.Instruction, ix) for ix in p.instructions],[convert(Braket.AbstractProgramResult, rt) for rt in p.results], [convert(Braket.Instruction, ix) for ix in p.basis_rotation_instructions])
 Base.convert(::Type{Braket.AbstractProgram}, p::BraketSimulator.OpenQasmProgram) = Braket.OpenQasmProgram(Braket.braketSchemaHeader("braket.ir.openqasm.program", "1"), p.source, p.inputs)
 
+function Base.convert(::Type{Braket.Circuit}, c::BraketSimulator.Circuit) 
+    ixs = [convert(Braket.Instruction, ix) for ix in c.instructions]
+    rts = isempty(c.result_types) ? Braket.Result[] : [convert(Braket.Result, rt) for rt in c.result_types]
+    brs = [convert(Braket.Instruction, ix) for ix in c.basis_rotation_instructions] 
+    Braket.Circuit(Braket.Moments(), ixs, rts, brs)
```

Comment:
It might be better to use `add_instruction!` for the instructions to populate the moments, so that the returned `Braket.Circuit` can have noise models applied correctly.